# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../perl/use_db3.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Force perl to use bdb33, compatibility reasons.

--- ./ext/DB_File/Makefile.PL.orig	2003-09-28 09:29:41.000000000 -0400
+++ ./ext/DB_File/Makefile.PL	2003-09-28 09:31:37.000000000 -0400
@@ -8,15 +8,15 @@
 # OS2 is a special case, so check for it now.
 my $OS2 = "-DOS2" if $Config{'osname'} eq 'os2' ;
 
-my $LIB = "-ldb" ;
+my $LIB = "-ldb-3" ;
 # so is win32
 $LIB = "-llibdb" if $^O eq 'MSWin32' ;
 
 WriteMakefile(
 	NAME 		=> 'DB_File',
-	LIBS 		=> ["-L/usr/local/lib $LIB"],
+	LIBS 		=> ["-L/usr/lib $LIB"],
         MAN3PODS        => {},         # Pods will be built by installman.
-	#INC		=> '-I/usr/local/include',
+	INC		=> '-I/usr/include/db3',
 	VERSION_FROM	=> 'DB_File.pm',
 	OBJECT		=> 'version$(OBJ_EXT) DB_File$(OBJ_EXT)',
 	XSPROTOARG	=> '-noprototypes',
